<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsKCM</class>
 <widget class="QWidget" name="SettingsKCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="6" column="0" rowspan="2" colspan="2">
    <widget class="KpkTransactionBar" name="transactionBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Update Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="intervalL">
        <property name="text">
         <string>Check for updates:</string>
        </property>
        <property name="buddy">
         <cstring>intervalCB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="intervalCB">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="autoInsL">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Automatically install:</string>
        </property>
        <property name="buddy">
         <cstring>autoCB</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KComboBox" name="autoCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="notifyUpdatesCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Notify when updates are available</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Global Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="autoConfirmCB">
        <property name="text">
         <string>Show confirmation dialog when installing or updating packages</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="appLauncherCB">
        <property name="text">
         <string>Show application launcher after installing new applications</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="originGB">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeView" name="originTV">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showOriginsCB">
        <property name="text">
         <string>&amp;Show origins of debug and development packages</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KpkTransactionBar</class>
   <extends>QWidget</extends>
   <header location="global">KpkTransactionBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>intervalCB</tabstop>
  <tabstop>autoCB</tabstop>
  <tabstop>notifyUpdatesCB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
